pyuvdata/utils.py

```diff
@@ -1490,7 +1509,9 @@ def ECEF_from_rotECEF(xyz, longitude):
     return rot_matrix.dot(xyz.T).T
 
 
-def ENU_from_ECEF(xyz, latitude, longitude, altitude, frame="ITRS"):
+def ENU_from_ECEF(
+    xyz, latitude, longitude, altitude, frame="ITRS", lunar_ellipsoid="SPHERE"
```

Comment:
Same comment here re: ellipsoid (note that there are several in utils here that need to be updated if you wanted to make that shift).